app/models/tag.rb

```diff
@@ -0,0 +1,3 @@
+class Tag < ApplicationRecord
+  belongs_to :post
```

Comment:
Should tags be a many to many relationship? This seems like it'll create a lot of duplication in the tags table.

Reply:
Good catch!  I'll make that change :)